cpu/kinetis_common/hwtimer_arch.c

```diff
@@ -27,8 +27,55 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+#define ENABLE_STATS (0)
+
+#if ENABLE_STATS
+/* Collect some statistics */
+struct {
+    unsigned int count_set_total;
```

Comment:
since there are no outside facing access to the statistics (you can only see them by using `print hwtimer_stats` inside GDB) I reasoned that it does not make sense to specify the bit size of them